tests/pytests/test_mex_drivers.py

```diff
@@ -0,0 +1,347 @@
+          "starting_detector_sample": 80,
+          "focal2pixel_lines": [
+            -7113.11359717265,
+            0.062856784318668,
+            142.857129028729
+          ],
+          "focal2pixel_samples": [
+            -0.778052433438109,
+            -142.857129028729,
+            0.062856784318668
+          ],
+          "optical_distortion": {
+            "radial": {
+              "coefficients": [
+                0.0,
+                0.0,
+                0.0
+              ]
+            }
+          },
+          "image_lines": 400,
+          "image_samples": 1288,
+          "name_platform": "MARS EXPRESS",
+          "name_sensor": "HIGH RESOLUTION STEREO CAMERA",
+          "reference_height": {
+            "maxheight": 1000,
+            "minheight": -1000,
+            "unit": "m"
+          },
+          "name_model": "USGS_ASTRO_LINE_SCANNER_SENSOR_MODEL",
+          "interpolation_method": "lagrange",
+          "line_scan_rate": [
+            [
+              0.5,
+              -94.88182842731476,
+              0.012800790786743165
+            ],
+            [
+              1.5,
+              -94.8690276145935,
+              0.012800790786743165
+            ],
+            [
+              15086.5,
+              101.82391116023064,
+              0.013227428436279297
+            ]
+          ],
+          "starting_ephemeris_time": 255744592.07217148,
+          "center_ephemeris_time": 255744693.90931007,
+          "t0_ephemeris": -101.83713859319687,
+          "dt_ephemeris": 40.734855437278746,
+          "t0_quaternion": -101.83713859319687,
+          "dt_quaternion": 40.734855437278746}
+
+@pytest.fixture(scope="module", autouse=True)
+def test_kernels():
+    kernels = get_image_kernels('h5270_0000_ir2')
+    updated_kernels, binary_kernels = convert_kernels(kernels)
+    spice.furnsh(updated_kernels)
+    yield updated_kernels
+    spice.unload(updated_kernels)
+    for kern in binary_kernels:
+        os.remove(kern)
+
+def test_mex_load(usgscsm_compare_dict):
+    label_file = get_image_label('h5270_0000_ir2')
+
+    with patch('ale.drivers.mex_drivers.MexHrscPds3NaifSpiceDriver.binary_ephemeris_times', \
+               new_callable=PropertyMock) as binary_ephemeris_times, \
+        patch('ale.drivers.mex_drivers.MexHrscPds3NaifSpiceDriver.binary_exposure_durations', \
+               new_callable=PropertyMock) as binary_exposure_durations, \
+        patch('ale.drivers.mex_drivers.MexHrscPds3NaifSpiceDriver.binary_lines', \
+               new_callable=PropertyMock) as binary_lines:
+        binary_ephemeris_times.return_value = [255744599.02748165, 255744599.04028246, 255744795.73322123]
+        binary_exposure_durations.return_value = [0.012800790786743165, 0.012800790786743165, 0.013227428436279297]
+        binary_lines.return_value = [0.5, 1.5, 15086.5]
+
+        driver = MexHrscPds3NaifSpiceDriver(label_file)
+        usgscsm_isd = to_usgscsm(driver)
```

Comment:
This should be calling load, not the driver and the formatter.

Reply:
When I use the load function, an exception is thrown. This is the error for trying to load the Mex driver:
```
  File "/home/kdlee/builds/ale/ale/drivers/__init__.py", line 69, in load
    with res as driver:
  File "/home/kdlee/builds/ale/ale/base/data_naif.py", line 17, in __enter__
    [spice.furnsh(k) for k in self.kernels]
TypeError: 'function' object is not iterable
```

Reply:
You need to add the test_kernels fixture to that test. If you don't add the fixture to the test, it just looks like a function to Python.